pyscal/pyscalcli.py

```diff
@@ -165,8 +166,10 @@ def main():
             slgof=args.slgof,
             family2=args.family2,
         )
-    except ValueError:
-        # If ValueErrors, error messages have already been printed
+    except (OSError, ValueError) as err:
+        if args.debug:
+            print("".join(traceback.format_tb(err.__traceback__)))
+        print(str(err))
```

Comment:
PR looks like a good change, but wouldnt you want this message on stderr? You could just `sys.exit(str(err))`

Reply:
Thanks, implemented.